querying-via-cli.md

```diff
@@ -127,13 +127,13 @@ In the `guac-data-main/docs/spdx/spdx_vuln.json` SBOM example, the URI would be
 This would result in the query being:
 
 ```bash
-guacone query vuln "https://anchore.com/syft/image/ghcr.io/guacsec/vul-image-latest-6fd9de7b-9bec-4ae7-99d9-4b5e5ef6b869"
+guacone query vuln "https://anchore.com/syft/image/ghcr.io/guacsec/vul-image-latest-6fd9de7b-9bec-4ae7-99d9-4b5e5ef6b869" "uri"
```

Comment:
Isn't this backwards? Shouldn't it be `guacone query vuln uri "https://anchore.com/..."`?

Reply:
I wrote the code in GUAC so that the args can be in any order. This means that the type (in this case "uri") can be before the input string, or after the input string.

Reply:
Oh, okay! In that case, I prefer the `<type> <input>` style over the `<input> <type>` style, but it'd be fine either way.

Reply:
@funnelfiasco, now that I think about it, I agree that `<type> <input>` is a bit more comprehensive than `<input> <type>`. So, I have changed the inputs to be in the format of `<type> <input>`. Thanks!